exact_test.go

```diff
@@ -1,44 +1,148 @@
 package exact_test
 
 import (
-	"fmt"
+	format "fmt"
+	"math/big"
 	"testing"
 
 	"github.com/madlambda/exact"
 )
 
-type Frac = exact.Frac // just because I'm lazy
+func newInt(t string) *big.Int {
+	v, _ := big.NewInt(0).SetString(t, 10)
+	return v
+}
+
+var (
+	sqrt11_3 = exact.NewFrac2(
```

Comment:
OMG, what is this ? =P

Reply:
the result of `sqrt(11/3)` kkkk
Idea?

Reply:
uaheuae I see...this one is hard to name indeed =P